src/zjs_modules.c

```diff
@@ -189,13 +187,18 @@ static ZJS_DECL_FUNC(native_require_handler)
 
     // Get the module name
     jerry_size_t module_size = jerry_get_utf8_string_size(argv[0]) + 1;
+    if (module_size > MAX_MODULE_STR_LEN) {
+        return RANGE_ERROR("module name too long");
+    }
+
     char module[module_size];
     zjs_copy_jstring(argv[0], module, &module_size);
 
     // Try each of the resolvers to see if we can find the requested module
     jerry_value_t result = jerryx_module_resolve(argv[0], resolvers, 3);
     if (jerry_value_has_error_flag(result)) {
-        ERR_PRINT("Couldn't load module %s\n", module);
+        //ERR_PRINT("Couldn't load module %s\n", module);
```

Comment:
How about make this a DBG_PRINT, module name would still be useful for debugging.